[2021-05-09 05:29:58,233] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: helloworld_dag.t4 2021-05-08T06:52:16.364066+00:00 [queued]>
[2021-05-09 05:29:58,247] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: helloworld_dag.t4 2021-05-08T06:52:16.364066+00:00 [queued]>
[2021-05-09 05:29:58,248] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2021-05-09 05:29:58,250] {taskinstance.py:1069} INFO - Starting attempt 1 of 1
[2021-05-09 05:29:58,251] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2021-05-09 05:29:58,275] {taskinstance.py:1089} INFO - Executing <Task(MyOperator): t4> on 2021-05-08T06:52:16.364066+00:00
[2021-05-09 05:29:58,281] {standard_task_runner.py:52} INFO - Started process 58 to run task
[2021-05-09 05:29:58,287] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'helloworld_dag', 't4', '2021-05-08T06:52:16.364066+00:00', '--job-id', '120', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/helloworld.py', '--cfg-path', '/tmp/tmpxkdj_xv4', '--error-file', '/tmp/tmpkk8xls6m']
[2021-05-09 05:29:58,289] {standard_task_runner.py:77} INFO - Job 120: Subtask t4
[2021-05-09 05:29:58,370] {logging_mixin.py:104} INFO - Running <TaskInstance: helloworld_dag.t4 2021-05-08T06:52:16.364066+00:00 [running]> on host 2900f64db589
[2021-05-09 05:29:58,444] {taskinstance.py:1283} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=helloworld_dag
AIRFLOW_CTX_TASK_ID=t4
AIRFLOW_CTX_EXECUTION_DATE=2021-05-08T06:52:16.364066+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-05-08T06:52:16.364066+00:00
[2021-05-09 05:29:58,447] {logging_mixin.py:104} INFO - Hello abc
[2021-05-09 05:29:58,487] {taskinstance.py:1192} INFO - Marking task as SUCCESS. dag_id=helloworld_dag, task_id=t4, execution_date=20210508T065216, start_date=20210509T052958, end_date=20210509T052958
[2021-05-09 05:29:58,534] {taskinstance.py:1246} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-05-09 05:29:58,586] {local_task_job.py:146} INFO - Task exited with return code 0
